community-handbook/community-report.md

```diff
@@ -0,0 +1,33 @@
+# Community Report
+
+The CHAOSS Community Report is a free service that the CHAOSS community provides to open source communities.
+
+## Goals
+
+* Create awareness for the CHAOSS project
+* Showcase the CHAOSS metrics and software
+* Get more communities interested in CHAOSS metrics and software
+* Grow the CHAOSS community
+
+## Roles
+
+The Communit Report processes have the following roles:
+
+* Requester -- Someone asking for a Community Report
+* Coordinator -- The CHAOSS contact person coordianting the creation and distribution of Community Reports
+
+## Requesting a Community Report
+
+Anyone can fill out a form on the CHAOSS.community website. The form submits to a Linux Foundation system. 
```

Comment:
and the form is forwarded to essential CHAOSS community members for processing.

Reply:
When I was reviewing the community report draft a thought came as to how we will get the project logo to be included in the community report. Is the project logo available on GitHub or we ask for the logo from the requester at the time of submission of the request. Furthermore, what information we require from the requester. @GeorgLink  and @sgoggins  if you can guide what information you require to generate these graphs like owner/repo or owner/list of repos or any other information than I can draft a request form.